modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -66,15 +68,25 @@ def clear_appointments_cache
       end
 
       def fetch_appointments(validated_params)
-        appointments = appointments_cache_interface.fetch_appointments(
+        appointments, failures = appointments_cache_interface.fetch_appointments(
           user: @current_user, start_date: validated_params[:start_date], end_date: validated_params[:end_date],
           fetch_cache: validated_params[:use_cache]
         )
 
         appointments.filter! { |appt| appt.is_pending == false } unless include_pending?(validated_params)
         appointments.reverse! if validated_params[:reverse_sort]
 
-        appointments
+        [appointments, failures]
+      end
+
+      def partial_errors(failures)
+        if failures&.any?
+          StatsD.increment('mobile.appointments.partial')
```

Comment:
I'm not sure I see the value in this metric. Someone recently removed a lot of our statsd incrementers for reasons I don't fully understand yet (apparently they didn't work correctly), so I think it might be best to remove this for now. https://github.com/department-of-veterans-affairs/vets-api/pull/12426

I do think it could be useful to do some rails logging here to give us a better idea of what failure types exist.

Reply:
I probably misunderstood this from the statsd logging requirement listed under expected outcome. I changed it to Rails logger